### DISCUSSION TITLE
Question regarded certain images

### DISCUSSION THREAD

**Discussion by Icythewolf** [silveredgold/beta-protection#49] [General]
Hi there, Bit of a issue question but mostly wanting to ask.
Does the programm have difficulty with drawn images?
A few sites and or images I wanted it to censor just showed the error sign emedietly.
With a whole site being nothing but the error?

**Comment by silveredgold** [comment]
That won't be a case of the art style (i.e. drawn vs non-drawn). If it's showing an error image, that means your current backend couldn't understand/process/censor the image. Without knowing what images cause that, or what backend you're using, it's very hard to say what could be causing that.
